### PR TITLE
fixed names generator generate invalid names and annotations are crea…

### PR DESCRIPTION
**Pull request by roironn**
…ted without a key

### CODE REVIEW

File: packages/dummy-adapter/src/generator.ts [hunk 1]
```diff
@@ -296,13 +296,23 @@ export const generateElements = (params: GeneratorParams): Element[] => {
...
   const generateFields = (): Record<string, FieldDefinition> => Object.fromEntries(
     arrayOf(
       normalRandom(defaultParams.fieldsNumMean, defaultParams.fieldsNumStd),
       () => {
         const name = getName()
         const fieldType = getFieldType(true)
-        return [name, { type: fieldType, annotations: generateValue(fieldType) }]
+        return [name, { type: fieldType,
+          annotations:
+          generateAnnotations(fieldType.annotationTypes) }]
```
Comment (netama) [inline packages/dummy-adapter/src/generator.ts:315]: why was this change needed? just for consistency or is it solving something?

**Review by netama — APPROVED** [review]

File: packages/dummy-adapter/src/generator.ts (hunk 1 above)
Comment (roironn) [reply packages/dummy-adapter/src/generator.ts:315]: Before the change the annotations were generated as a single value, which is wrong since the annotation key should hold a map, no a string...
